client/client.js

```diff
@@ -28,6 +26,8 @@ $('form').submit(function(){
   replaceMessage("<3", "❤️");
   replaceMessage(":)", "😃");
   replaceMessage("ratm", '<a href="https://www.youtube.com/watch?v=wauzrPn0cfg">ratm</a>');
+  replaceMessage("Ratm", '<a href="https://www.youtube.com/watch?v=wauzrPn0cfg">Ratm</a>');
+  replaceMessage("RATM", '<a href="https://www.youtube.com/watch?v=wauzrPn0cfg">RATM</a>');
```

Comment:
you could consider doing the search on a lowercase `message`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase

Then you wouldn't need to test so many cases